dids/src/main/kotlin/web5/sdk/dids/DidWeb.kt

```diff
@@ -0,0 +1,131 @@
+package web5.sdk.dids
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import foundation.identity.did.DID
+import foundation.identity.did.DIDDocument
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.HttpClientEngine
+import io.ktor.client.engine.cio.CIO
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.client.request.get
+import io.ktor.client.statement.bodyAsText
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import io.ktor.http.isSuccess
+import io.ktor.serialization.jackson.jackson
+import kotlinx.coroutines.runBlocking
+import web5.sdk.crypto.KeyManager
+import java.net.URLDecoder
+import java.nio.charset.StandardCharsets
+
+/**
+ * Provides a specific implementation for creating and resolving "did:web" method Decentralized Identifiers (DIDs).
+ *
+ * A "did:web" DID is an implementation that uses the web domains existing reputation system. More details can be
+ * read in https://w3c-ccg.github.io/did-method-web/
+ *
+ * @property uri The URI of the "did:web" which conforms to the DID standard.
+ * @property keyManager A [KeyManager] instance utilized to manage the cryptographic keys associated with the DID.
+ *
+ * ### Usage Example:
+ * ```kotlin
+ * val keyManager = InMemoryKeyManager()
+ * val did = StatefulWebDid("did:web:tbd.website", keyManager)
+ * ```
+ */
+public class StatefulWebDid(
```

Comment:
Can we just call it DidWeb? 
Even though it is a mutant object with a keyManager, I think its ok to call it didWeb and the didWeb.uri is the actual didWeb